src/server_manager/messages/iw.json

```diff
@@ -0,0 +1,272 @@
+{
```

Comment:
Please don't include `iw` as well, it is identical to `he` which is already included. Here is [more information](https://source.corp.google.com/piper///depot/google3/jigsaw/firehook/outline/BUILD;l=25).

Reply:
done